packages/nextclade-web/src/io/fetchSingleDatasetAuspice.ts

```diff
@@ -1,13 +1,24 @@
+import { isEmpty } from 'lodash'
 import { attrStrMaybe, AuspiceTree, Dataset } from 'src/types'
 import { removeTrailingSlash } from 'src/io/url'
-import { axiosFetch } from 'src/io/axiosFetch'
+import { axiosFetch, axiosFetchOrUndefined } from 'src/io/axiosFetch'
 
 export async function fetchSingleDatasetAuspice(datasetJsonUrl_: string) {
   const datasetJsonUrl = removeTrailingSlash(datasetJsonUrl_)
 
   const auspiceJson = await axiosFetch<AuspiceTree>(datasetJsonUrl, {
     headers: { Accept: 'application/json, text/plain, */*' },
```

Comment:
Since using our media type in the sidecar fetch below, might as well use our main dataset JSON media type here too, with quality factors to rank preferences, e.g.

```suggestion
    headers: { Accept: 'application/vnd.nextstrain.dataset.main+json;q=1, application/json;q=0.9, text/plain;q=0.8, */*;q=0.1' },
```